### PR TITLE
api: added defaults

### PR DESCRIPTION
**Pull request by aramikm**
# Problem

Related to #491 

# Solution
- fixed a previously introduced bug related to link assets

with @pairperson1

## Steps to Verify:

1. run any of the apis
2. open swagger
3. try to submit a bigger than supported json

### CODE REVIEW

File: apps/graph-api/src/main.ts [hunk 1]
```diff
@@ -37,7 +40,15 @@ async function bootstrap() {
...
   try {
     app.enableShutdownHooks();
-    app.useGlobalPipes(new ValidationPipe());
+    app.useGlobalPipes(
+      new ValidationPipe({
+        whitelist: true,
+        transform: true,
+      }),
+    );
+    // TODO: should get replaced with the config
```
Comment (aramikm) [inline apps/graph-api/src/main.ts:49]: seems like config in graph api is not refactored yet!

File: docker-compose.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ x-common-environment: &common-environment
 
 x-content-publishing-env: &content-publishing-env
   START_PROCESS: content-publishing-api
-  FILE_UPLOAD_MAX_SIZE_IN_BYTES: 2000000000
+  FILE_UPLOAD_MAX_SIZE_IN_BYTES: 500000000
```
Comment (aramikm) [inline docker-compose.yaml:26]: it seems like we can only upload files up to 500MB based on current setup and redis

File: openapi-specs/content-publishing.openapi.json [hunk 3]
```diff
@@ -334,89 +334,6 @@
           "health"
         ]
       }
-    },
```
Comment (aramikm) [inline openapi-specs/content-publishing.openapi.json:337]: for some reason my IDE likes a lot to apply this change.

File: libs/types/src/dtos/content-publishing/activity.dto.ts [hunk 4]
```diff
@@ -190,7 +191,16 @@ export class TagDto {
 }
 
 export class AssetDto {
-  @ValidateIf((o) => o.type !== AttachmentType.LINK)
+  @ApiPropertyOptional({
+    description: 'Determines that if this asset is a link',
```
Comment (mattheworris) [inline libs/types/src/dtos/content-publishing/activity.dto.ts:195]:
```suggestion
    description: 'Determines if this asset is a link',
```

**Review by mattheworris — APPROVED** [review]
- Read through changes, everything makes sense to me, nice work.
- Did not test
🚢 it!

**Pull request merged**